src/tools/rbd_mirror/image_replayer/snapshot/Replayer.cc

```diff
@@ -880,7 +947,7 @@ void Replayer<I>::handle_get_remote_image_state(int r) {
     return;
   }
 
-  create_non_primary_snapshot();
+  create_group_snap_start();
```

Comment:
A better name/approach here  ?  I'm not in favour of the image replayer function name explicitly performing a higher level group snap creation.